hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenUnfreezeAccountHandler.java

```diff
@@ -130,10 +131,7 @@ private TokenRelation validateSemantics(
         validateTrue(tokenRel != null, TOKEN_NOT_ASSOCIATED_TO_ACCOUNT);
 
         // Check that the token itself exists and is valid
-        final var token = tokenStore.get(tokenId);
-        validateTrue(token != null, INVALID_TOKEN_ID);
-        validateFalse(token.deleted(), TOKEN_WAS_DELETED);
-
+        var token = TokenHandlerHelper.getIfUsable(tokenId, tokenStore);
```

Comment:
My mistake, I was thinking 'freeze' and 'pause' were the same operation. `getIfUsable` should be the correct choice here (as it's written) 👍